hack/generator/pkg/astmodel/testdata/SetOptionalStringFromOptionalString-Direct.golden

```diff
@@ -13,7 +13,8 @@ type Person struct {
 func (person Person) ConvertFrom(source *vHub.Person) error {
 
 	// name
-	person.name = source.name
+	name := *source.name
```

Comment:
This will panic if `source.name` is nil - wrap it in an if? (Same for all of the optional-optional conversions I think, it's just clearest here.)

Reply:
Good catch! Thanks.

![](https://media.giphy.com/media/l0Iy69RBwtdmvwkIo/giphy.gif)